### DISCUSSION TITLE
BlueSCSI on Sun systems

### DISCUSSION THREAD

**Discussion by Atm200** [BlueSCSI/BlueSCSI-v2#224] [Help] [answered]
Hi, i recently got a BlueSCSI for my Sun Sparcstation and i having issues with system inastallation on it. The board itself works fine, i made a dump of Sparcstation HDD but solaris installer throws an error trying to make file system, i can make and edit disk label but cant make file system. Here photo of error, also nuber of sectors in error dont change regardless of image file size (sorry for crappy photo).
Thanks for your help

![DSC_2116](https://github.com/user-attachments/assets/21ef2e4c-28dc-44de-93f8-0703749adbbe)

**Comment by erichelgeson** [comment]
Hi there - I'm not an expert in Sun systems so I cant tell you what that error means - though I know there are some BlueSCSI Sun users in our Discord. 

I'd first have you go completely through the [troubleshooting guide](https://github.com/blueSCSI/blueSCSI-v2/wiki/Troubleshooting#what-to-try-first) and re-check the [compatibility](https://github.com/blueSCSI/blueSCSI-v2/wiki/Compatibility#sun) page to rule out the easy things first - after that please provide a debug log (the machine will be slower while debug is active)

**Reply by Atm200** [reply]
The board works fine in my other non sun machine, i start to suspect that problem is in default disk geometry and i going try to play with it for a bit. Here's debug log. HDD on id3, command for file system creation was run twice if it give some help.
[log.txt](https://github.com/user-attachments/files/18286732/log.txt)

**Reply by Atm200** [reply, marked as answer]
Problem was in disk geomertry. I change sectors per track to 60 and heads per cylinder to 245 and everything start to work as it should

**Reply by coolcrm** [reply]
Hello, could you kindly share you ini file and terminator position?

**Reply by Atm200** [reply]
Sure, catch. Termination is on
`[SCSI]
Quirks=0
InitiatorMode=0
[SCSI4]
SectorsPerTrack=248
HeadsPerCylinder=19
[SCSI3]
Vendor = "FUJITSU"
Product = "MAB3091S SUN9.0G"
Version = "1.0"
Serial = "0123456789ABCDEF"
SectorsPerTrack=133
HeadsPerCylinder=27`

**Reply by coolcrm** [reply]
Thanks a lot!!
